man/man7/zpoolconcepts.7

```diff
@@ -66,13 +66,14 @@ A mirror of two or more devices.
 Data is replicated in an identical fashion across all components of a mirror.
 A mirror with
 .Em N No disks of size Em X No can hold Em X No bytes and can withstand Em N-1
-devices failing without losing data.
+devices failing, without losing data.
```

Comment:
Could you please point me as non-native speaker on reasoning here? Looks like old variant was ok.

Reply:
Not terribly important. It's a prepositional phrase, where a comma would sometimes be used. It doesn't affect the clarity of the sentence, really. Could be considered stylistic, and is not strictly required. I was trying to make it more consistent with the rest of the document.

Reply:
It's probably one of the least important commas I inserted in the whole thing.